doc/mapflags.txt

```diff
@@ -396,3 +396,10 @@ in '/conf/channels.conf'.
 Disables MVP tombs from appearing on a map.
 
 ---------------------------------------
+
+*hidehpbar
+
+Hides monster's HP bar on a map.
+Ignores config value of 'monster_hp_bars_info'.
```

Comment:
doesn't matter, because it's not decide to show hp bar.
I put after battle config first because "I don't need to check mapflag if the feature itself is disabled"